CMakeLists.txt

```diff
@@ -92,5 +92,4 @@ if(("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang") OR
 endif()
 
 add_subdirectory(src)
-
-install(FILES LICENSE DESTINATION .)
```

Comment:
Current "install" steps are useful on Windows to create a release package. The copy all binaries, python api, examples, license and 3rdpartyy (protobuf/python) to "C:\Program Files\BigARTM". This makes it easy to prepare release drops for Windows. I only pack with 7z and upload to github :) Is it OK to apply your changes for all OS except windows, and keep the steps on windows as today?